src/components/ReportLoader.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { CircularProgress } from '@dhis2/d2-ui-core'
+import isEmpty from 'lodash.isempty'
+import './ReportLoader/styles.css'
+
+const ReportLoader = ({ isLoading, content, children }) => {
```

Comment:
I'm keeping this name, purely because of the fact that it does do one quirky thing: it checks if `content` is an empty object, and if so, it renders nothing. As such it is implied that is fed some reportData state.